chart/examples/minikube/values.yaml

```diff
@@ -96,8 +96,10 @@ search:
 app:
   # app.memory -- Configures the amount of memory Java can use
   memory: 256M
-  # app.memory -- Configures runtime mode for fusionauth. Should be 'development' or 'production'
+  # app.runtimeMode -- Configures runtime mode for fusionauth. Should be 'development' or 'production'
   runtimeMode: development
+  # app.silentMode -- Configures silent mode for fusionauth. Should be 'true' or 'false'
+  silentMode: "true"
```

Comment:
Please use boolean instead of a string. Should `true` be the default?

Reply:
I felt the minikube-example was for local testing purposes only and therefore transparent silent mode db migrations are the easy route. I don't have strong opinions on this either way and will change it if you feel it should be `false`.

Reply:
I don't have strong opinions either. Let's leave it as `true` for now.